### ISSUE TITLE
Example use? `(0 , sessions_1.createSessionStorage) is not a function` error

### ISSUE THREAD

**Issue by edwin177** [DAlperin/remix-redis-session#1]
I was looking for an equivalent to express-session and connect-redis for Remix and remix-auth. Is that what you're aiming for? If so, would you share a working example?

I tried replacing the `createCookieSessionStorage` call with `createRedisSessionStorage` and that doesn't seem to be enough. I'm getting the following error:

```
/home/xxx/yyy/node_modules/remix-redis-session/dist/index.js:40
    return (0, sessions_1.createSessionStorage)({
                                               ^
TypeError: (0 , sessions_1.createSessionStorage) is not a function
    at createRedisSessionStorage (/home/xxx/yyy/node_modules/remix-redis-session/dist/index.js:40:48)
    at Object.<anonymous> (/home/xxx/yyy/src/remix/app/auth/session.server.ts:9:29)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47
```

**Comment by DAlperin** [comment]
Can you copy in your full createRedisSessionStorage call?

**Comment by edwin177** [comment]
```
export let sessionStorage = createRedisSessionStorage({
  cookie: {
    name: 'cookie-name', // use any name you want here
    domain: 'some-domain.com',
    maxAge: 1000 * 60 * SOME_MINS,
    sameSite: 'lax', // this helps with CSRF
    path: '/', // remember to add this so the cookie will work in all routes
    httpOnly: true, // for security reasons, make this cookie http only
    secrets: ['somesecret'], // replace this with an actual secret
    secure: env !== 'development' && env !== 'test' // enable this in prod only
  },
  options: {
    redisClient: rdb.getClient()
  }
});
```

`rdb.getClient()` returns a redis client created using [redis](https://github.com/redis/node-redis).

**Comment by DAlperin** [comment]
Could you try creating the client with ioredis? That probably... Isn't the problem but worth a shot

**Comment by DAlperin** [comment]
I should probably specify that this package prefers ioredis at the moment in the README

**Comment by DAlperin** [comment]
Also what version of remix are you using? I was chatting with @mjackson the other day and he informed me that the API this relies on is unstable. Maybe he has updated it since last we talked.

**Comment by DAlperin** [comment]
Oh that is definitely it. I just upgraded to 1.3.3 and am hitting the same error. Ok give me a minute.

**Comment by DAlperin** [comment]
@edwin177 thanks for the report! Fixed in version 0.0.9. If you just upgrade remix-redis-session it should work fine.
